### PR TITLE
New UI for iOS

### PR DESCRIPTION
**Pull request by mik-koroteev**

### CODE REVIEW

**Comment by ghazel** [comment]
This does not seem to be compatible with Dark Mode. I get a white screen with the button, and no text.

File: ios/NewNode VPN/NewNode VPN/Colors.xcassets/gradient_off1.colorset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          "value" : "dark"
+        }
+      ],
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "1.000",
+          "blue" : "1.000",
+          "green" : "1.000",
+          "red" : "1.000"
```
Comment (ghazel) [inline ios/NewNode VPN/NewNode VPN/Colors.xcassets/gradient_off1.colorset/Contents.json]: These cause the app to appear all white in Dark Mode. (all of the colors seem to have this problem)

**Review by ghazel — CHANGES_REQUESTED** [review]
This does not seem to be compatible with Dark Mode. I get a white screen with the button, and no text.

It needs to function in Dark Mode (even if it looks the same).

File: ios/NewNode VPN/NewNode VPN/Colors.xcassets/gradient_off1.colorset/Contents.json (hunk 1 above)
Comment (mik-koroteev) [reply ios/NewNode VPN/NewNode VPN/Colors.xcassets/gradient_off1.colorset/Contents.json]: You are right. Fixed in 921d78fc072ea322b3b01999f13130956a7d9754. I also inverted colors for the Info view in Dark Mode.

**Review by ghazel — APPROVED** [review]

**Pull request merged**